tests/e2e/manifest/reset.sh

```
@@ -2,16 +2,13 @@
 
 # NOTE: We are taking the argument as the path to the manifest directory.
 # This is needed as the script is called from multiple places and has different paths.
-# Maybe in future we can fix this, but for now this is fine. 
+# Maybe in future we can fix this, but for now this is fine.
 
 # Apply ElastiService
 kubectl apply -f  $1/target-elastiservice.yaml -n target
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)